src/modules/elements/label-with-tooltip/element.js

```diff
@@ -17,10 +17,10 @@ const LabelWithTooltip = ( {
 	className,
 	forId,
 	isLabel,
-	label,
+	label = '',
 	tooltipDisabled,
 	tooltipLabel,
-	tooltipPosition,
+	tooltipPosition = 'top right',
 	tooltipText,
 	delay,
```

Comment:
Ne default delay?

Reply:
No change of code here, just updating it to avoid the default properties issues.